README.md

```diff
@@ -1,2 +1,44 @@
-# Mobile-first-D3
-Mobile First approach
+# PORTFOLIO - MICROVERSE TASK
+
+> This a "mobile-first" approach, that gives more priorities in developing mobile version first and perfecting the mobile user experience. Due to the fact that there are rising numbers of mobile users as compared to the desktop users.
+
+## Built With
+
+- Html,Css,
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+All you need VScompiler or a compiler of your choice.
+
+### Setup
+
+You will need to clone the repo and open the html with live server.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](felixDev22)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com).
+
+## Show your support
+
+Give a ⭐️ ⭐️ if you like this project!.
+
+## Acknowledgments
+
+- Inspired by Microverse in the spirit of equiping developers with the right tools.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL]
- Let's add the missing license here to make our readme more professional.